example.css

```diff
@@ -1,6 +1,28 @@
-/*
-  Examples of good css:
-   - Naming conventions
-   - Codestyle
-   - BEM dialect
-*/
+<!DOCTYPE html>
```

Comment:
It's css file for examples of naming conversions, not for "how to write a simple css selector" examples.